wordpress-mobile/WordPress-iOS
# Contact Support: Dynamic Type fixes

Related to: #21557

To support dynamic type, enabling `-webkit-text-size-adjust` didn't work. It only caused the text size to increase when switching to a horizontal orientation.

What worked effectively, is setting the font to a specific apple-system font type. 

I chose to set the regular font to `apple-system-body` and smaller labels to `apple-system-footnote` which would follow the default font size of the system.

I also positioned suggested questions relatively, otherwise increasing font size causes initial questions to appear on top of the first message, rather than below it.

https://github.com/wordpress-mobile/WordPress-iOS/assets/4062343/6d3093db-78c8-4f02-8779-2dad1ae59386

## To test:

1. Open Help & Support
2. Increase / decrease font size
3. Confirm suggested questions, initial message, and input field fonts and container react to size changes
4. Ask a question
5. increase/decrease font size
6. Confirm message, sources, contact support button, and input field fonts and container react to size changes

## Regression Notes
1. Potential unintended areas of impact

Couldn't identify

2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:


UI Changes testing checklist:
